clmm/support/sampler.py

```diff
@@ -4,32 +4,129 @@
 """
 
 
-def sciopt(model_to_shear_profile, logm_0, args) :
-    ''' Uses scipy optimize minimize to output the peak'''
+def sciopt(model_to_shear_profile, logm_0, **kwargs):
+    r''' Uses scipy optimize minimize to output the peak
+    
+    Parameters
+    ----------
+    model_to_shear_profile : callable
+        The objective function to be minimized.
+        ``model_to_shear_profile(x, *args) -> float``
+        where ``x`` is an 1-D array with shape (n,) and ``args``
+        is a tuple of the fixed parameters needed to completely
+        specify the function.
+    logm_0 : ndarray, shape (n,)
+        Initial guess. Array of real elements of size (n,),
+        where 'n' is the number of independent variables.
+    kwargs :
+        Other optional keyword arguments passed to `minimize`.
+        Please check the `scipy` documentaion for information on default values and methods.
+        
+    Returns
+    -------
+    `x`` : array
```

Comment:
Not sure if the quotation marks around x are necessary. They do not look good in the compiled HTML file.

Reply:
Ahhh! Good catch!

Reply:
Thank you!

Reply:
Yeah, I'm happy to approve the merge as soon as the \`x\`\` is fixed.

Reply:
@aimalz I can fix this very quickly. But I am not sure if it should be just an x, or there are two double backquotes (or two single backquotes) around it.

Reply:
Just checked, it should be just an x for the documentation to render properly